.storybook/config.js

```diff
@@ -1,7 +1,7 @@
 import { configure } from '@storybook/react';
 
 function loadStories() {
-  require('../src/stories');
+  require('../src/storybook');
```

Comment:
Sorry, I didn't mean we should just scope the folders. I mean't to decouple storybook from src completely. So the stories should live in the `stories` folder as a sibling to `src`.

```
- package.json
- src/
- stories/
- .storybook/
```

This repo is a good example. https://github.com/airbnb/react-dates

Reply:
That might not be the move. I have seen plenty of repos that keep the stories in the `src`. Following our existing convention of `__test__` being in `src` which utilizes the components `stories` should also be in `src` as they too utilize components.

Further-more:
Here is what the official doc says:

"There’s no hard and fast rule for this. But, keeping stories close to your components is a good idea.

For example, let’s say your UI components live in a directory called: src/components. Then you can write stories inside the src/stories directory.

This is just one way to do that. You can always edit your storybook config file and ask it to load stories from anywhere you want."

https://storybook.js.org/basics/writing-stories/

Reply:
Our tests aren't in src. And I personally prefer things to be decoupled as much as possible. In fact, since our tests are decoupled outside of src, it would make more sense to decouple the stories too.

Reply:
https://github.com/MidnightJabber/foodie-ui/issues/47